### ISSUE TITLE
change from submodule to subtree

### ISSUE THREAD

**Issue by chiarcos** [cdli-gh/ETCSANS#1]
On Ubuntu 20.04.3, I got an authorization error when pushing from a repo checked out with `--recurse-submodules` (no changes in the submodules). It works seamlessly, when checking out without submodules and then applying, committing and pushing the same changes. While it is not clear why this is happening, this calls for a shift from `submodule` to `subtree`.

**Comment by epageperron** [comment]
So you think it's related to your rights in the org?

**Comment by chiarcos** [comment]
No, this is propagation of authentification information to submodules, and with the recent change to PATs, this has become a bit of a pain. Cf. https://medium.com/@alexander.sirenko/using-github-access-token-with-submodules-5038b6d639e8. Subtree shouldn't have these issues as it creates a local copy that behaves like any other repo content.
